include/swift/SIL/SILAllocated.h

```diff
@@ -28,8 +28,10 @@ class SILAllocated {
   void *operator new(size_t) = delete;
   void *operator new[](size_t) = delete;
 
-  /// Disable non-placement delete.
+  // Work around MSVC error: attempting to reference a deleted function.
+#if !defined(_MSC_VER) || defined(__clang__)
```

Comment:
I'm not really sure what causes these kinds of errors - there seem to be plenty of them! Is Clang or MSVC wrong?